<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_filePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>831</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_brightness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>Absolute brightness</string>
     </property>
     <item>
      <property name="text">
       <string>Absolute brightness</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Perceived brightness</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R,G,B as separate channels</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Which way should changes in brightness be determined</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>20</width>
       <height>381</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size of frame span to be considered in analysis. (More = Slow)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_format">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>video frames</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>milliseconds (s/1000)</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="spinBox_frameSpan">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QPushButton" name="button_applySettings">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>360</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Apply changes and process videofile with new settings</string>
     </property>
     <property name="text">
      <string>APPLY</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_isResetForced">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Will remove any cached data such as brightness information on each frame and scan the videofile again instead of reusing the information</string>
     </property>
     <property name="text">
      <string>Delete cached data</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Try to cache video files up to what file size (use with caution)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_filesize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gigabyte</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>145</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lasts. Limited to size of frame span. (More=Slow but fancy)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_plotMaxColors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of color shadings to show how long optical flickering</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Colors in graph</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_yLim">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>160</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>120</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Graphic settings - in case your analysis is a (fancy) mess</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Middle value at:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_colorBorderValue">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>60</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>0.990000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.800000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="button_applyGraphics">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>190</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Apply changes and process videofile with new settings</string>
     </property>
     <property name="text">
      <string>APPLY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y-Axis max value</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>220</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="button_processFile">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>280</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Apply changes and process videofile with new settings</string>
     </property>
     <property name="text">
      <string>Re-scan video file and start new analysis (forced)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_processData">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>320</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Apply changes and process videofile with new settings</string>
     </property>
     <property name="text">
      <string>Re-process all collected data on video file (forced)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_drawGraph">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>360</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Apply changes and process videofile with new settings</string>
     </property>
     <property name="text">
      <string>Re-draw graph using given results (forced)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>240</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Admin Controls, use with caution. Directly affect the behaviour</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>255</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>and could break everything but may also save you some time :)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_color1">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>85</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>#FF7000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>85</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gradient Color 1/3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gradient Color 2/3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>135</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gradient Color 3/3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_color2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>110</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>#FF0000</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_color3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>135</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>#000000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Color &amp; opacity of data plots are set by a 3-color gradient:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>20</width>
       <height>381</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>401</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_progress">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready, please select file...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_load">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>1</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Apply changes and process videofile with new settings</string>
   </property>
   <property name="text">
    <string>LOAD</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../shutterstock_695907658_klein.qrc"/>
 </resources>
 <connections/>
</ui>
